### ISSUE TITLE
Unable to import games requiring commandline, only workaround yields the oddest error I've encountered yet: different TITLE fields depending on whether imported as a singleton or en-masse

### ISSUE THREAD

**Issue by davejames3282** [JosefNemec/Playnite#3535]
### Bug Description

Background behavior (or lackthereof) leading to Concrete Error: Unless I am missing something extremely obvious, there are no script features/commandlines/methods to import a large amount of games that launch with the format:

'emulator/path/emulator.exe SystemName "RomName"

which is used by more high end systems and is certainly mandatory for ANY system that employs Fades, Loading Screens, Overlays, and the most basic of features to prevent the horrid Windows operating system from taking focus from the emulator window, abruptly flashing the window on the screen prior to the game being loaded, windows visibly changing sizes including displaying unsightly toolbars with minimize/close functions on them.

Unfortunately rendering PlayNite as nothing more than a simple GUI Shortcut Launcher.

Examples include:

C:\RL\RocketLauncher.EXE Nintendo 64 "Super Mario 64"
C:\HyperSpin\Hyperlaunch.EXE Sega Genesis "Sonic The Hedgehog"
C:\Memu\Memu86.EXE Sony PlayStation 2 "Grand Theft Auto III"
etc. etc.

Essentially anything with a clean pass.

Error during Workaround: **The workaround for this would be creating thousands of thousands of LNK shortcuts that run those command lines, however even if the end-user were to find spending hours making LNK files (or more likely running yet another 3rd party script to do that for them like HyperLaunch) that feature is BROKEN as importing them by mass IGNORES THE FILENAME and makes the title the file name something I've never seen in the file itself... thousands upon thousands of times (whereas importing the SAME FILE INDIVIDUALLY yields PlayNite titling the game with the proper title of the actual filename...)**

IMPORTANT BEHAVIOR: Files should auto-titled the SAME whether imported via mass scan or individually added... though that's just a workaround anyways, albeit an extremely convoluted workaround that simply does not work do to that altered behavior of broken TITLE importing.

Query: how long would it take to import the entire North American NES Library one...by one... by one... by one?

### To Reproduce

1. Import lnk en-masse via "Scan Folder" through attached scan feature and see an array of horrid singular title for multiple games
2. Import ONE of the SAME FILES individually using the same "Scan Folder" by selecting Browse _(which limits the end user to One.Single.File.At.A.Time... yet gets the title correctly correlated with the only title field available... the filename. I have no idea where the en-mass "swell guy" name is coming from. Regardless should be the same behavior)_

### Diagnostics ID

d753c771-e2f0-422b-b827-7440fb766590

### Screenshots

![folder](https://github.com/JosefNemec/Playnite/assets/142783086/9dc66d1e-9038-4ab9-b076-ab13581c22ed)
![import via scan multiple files](https://github.com/JosefNemec/Playnite/assets/142783086/6eea8bcd-7b66-48a0-84a8-de5f067578e2)
![importing same file via scan browse](https://github.com/JosefNemec/Playnite/assets/142783086/e2db7822-d979-4429-96d9-4714ea557a7f)

**Comment by JosefNemec** [comment]
You should be using built emulation support for scenarios like this or create custom library plugin with bespoke game import behavior.

**Comment by JosefNemec** [comment]
> there are no script features/commandlines/methods to import a large amount of games that launch with the format

Can be done in multiple ways:
- configured as an emulator
- made as a library plugin
- script extension you can call on demand to import games

https://api.playnite.link/docs/

**Comment by JosefNemec** [comment]
Can you attach some lnk files that resulted in bug from the second screenshot?

**Comment by davejames3282** [comment]
Will re-read the documentation, but during my last attempt saw no way to pass through the PLATFORM followed by the TITLE post the executable. Will double check in the morning though. If you want to point me to the option perceived as most appropriate I will start there. 

As this FrontEnd has gotten a lot of buzz in the recent months, I've encountered countless similar postings on the RL Forums, Mame Community, Reddit, etc. but have not actually seen any posted instructional from users and/or developers which led me to believe it was either not supported or unable to be understood (which may be my case as I went over that doc several times to no success)

Unable to post the LNK Files directly so am attaching them as ZIP. If the concern is there is metadata or detail fields in the file(s) themselves, that was my first thought, but they were blank from Explorer's view.

[LNK Files.zip](https://github.com/JosefNemec/Playnite/files/12394242/LNK.Files.zip)
